src/codegen.cpp

```diff
@@ -427,7 +427,10 @@ void jl_dump_objfile(char *fname, int jit_model)
     std::string err;
     raw_fd_ostream OS(fname, err);
 #endif
+
+#ifndef LLVM37 // 3.7 simplified formatted output
     formatted_raw_ostream FOS(OS);
+#endif
```

Comment:
stylistically, I think this would be better with an else clause here, rather than copying code below. perhaps something like:
```
#else
auto FOS = OS;
#endif
```